portfolio/src/main/webapp/script.js

```diff
@@ -49,14 +49,14 @@ function makeList(){
     fetch("/data").then(response => response.json()).then((wordList) => {
         const buildList = document.getElementById("make-me");
         buildList.innerHTML = "";
-        buildList.appendChild(createli(wordList[0]));
-        buildList.appendChild(createli(wordList[1]));
-        buildList.appendChild(createli(wordList[2]));
+        for(let i = 0; i < wordList.length; i++){
```

Comment:
FYI in Javascript there are a few ways to do it easier than this.

1. `for(let word of words) { buildList.appendChild(createListItem(word)) }`
2. `wordList.forEach((word) => { buildList.appendChild(createListItem(word)) });`

I believe the first is preferred by the style guide.

Anyhow optional if you want to change it, rest looks good.